internal/platform/data/fixtures/rule.json

```diff
@@ -0,0 +1,37 @@
+{
+  "source_language": "es",
+  "target_language": "en",
+  "pattern": "VERB,ADV,ADJ",
+  "details": [
+    {
+      "type": "VERB",
+      "properties": {}
```

Comment:
This changed from 
{
      "type": "ADV",
      "properties": {}
}
to
{
      "tag": "ADV",
      "type": "X"
}

Reply:
I am going to change this in the next PR